src/FixedPairList.cpp

```diff
@@ -274,6 +282,7 @@ namespace espressopp {
       .def("add", pyAdd)
       .def("size", &FixedPairList::size)
       .def("getBonds",  &FixedPairList::getBonds)
+      .def("remove",  &FixedPairList::remove)
```

Comment:
@govarguz 
If you run ```remove()``` defined here then this object cannot be used anymore because there is no way to connect to signals again.
For me, separating this to two different methods, one ```connect/disconnect``` and second ```deleteAll``` is a clean approach.
@niktre what do you think?

btw, I checked and calling ```del fpl``` on Python side does not call destructor...